packages/react-native-otlp/src/__tests__/index.test.ts

```diff
@@ -100,6 +105,40 @@ describe("React Native OTLP", () => {
     });
   });
 
+  describe("should call `startNativeEmbraceSDK` (Android specific)", () => {
+    beforeAll(() => {
+      Platform.OS = "android";
+    });
+
+    it("if it receives the proper configuration (happy path)", async () => {
+      const otlpExporterConfig = {
+        logExporter: {
+          endpoint: "https://example.com/logs/v1",
+          headers: [
+            {key: "Authorization", token: "Bearer token"},
+            {key: "Authorization", token: "Bearer token:v2"},
```

Comment:
is it valid to have the same key twice? Possibly headers should be a dictionary actually, but can revisit that separately from this PR